src/InsertTagHandler.php

```diff
@@ -84,9 +74,15 @@ public function parseInsertTag(string $tag): string|null
         return $this->parser->parse((string) $record['replacement'], $tokens);
     }
 
-    /**
-     * Get the tag record.
-     */
+    private function dcaFormat(string $table, string $field, mixed $value): mixed
```

Comment:
Maybe add a comment about compatibility with Haste 4 and 5 ? Return value should always be a string imho (haven't checked the methods though, might need casting).